src/bin/pgcopydb/table-data.c

```diff
@@ -1182,12 +1183,23 @@ copydb_prepare_copy_query(CopyTableDataSpec *tableSpecs,
 
 		for (int i = 0; i < table->attributes.count; i++)
 		{
-			char *attname = table->attributes.array[i].attname;
+			SourceTableAttribute *attribute = &(table->attributes.array[i]);
+			char *attname = attribute->attname;
 
-			if (i > 0)
+			/* Generated columns cannot be used in COPY */
+			if (attribute->attisgenerated)
+			{
+				log_info("Skipping %s in COPY as it is a generated column", attname);
```

Comment:
Do we need that information at the INFO level? I suppose NOTICE would be preferable.